ember-client/shared/src/main/scala/org/http4s/ember/client/EmberClientBuilder.scala

```diff
@@ -116,7 +111,8 @@ final class EmberClientBuilder[F[_]: Async: Network] private (
   def withoutTLSContext: EmberClientBuilder[F] = copy(tlsContextOpt = None)
 
   /** Sets the `SocketGroup`, a group of TCP sockets to be used in connections. */
-  def withSocketGroup(sg: SocketGroup[F]): EmberClientBuilder[F] = copy(sgOpt = sg.some)
+  @deprecated("Explicit socket groups are no longer supported", "0.23.next")
+  def withSocketGroup(sg: SocketGroup[F]): EmberClientBuilder[F] = this
```

Comment:
Just calling out that this is a breaking change. Not sure how much this feature was used in the wild.